openfisca_us_data/datasets/cps/cps.py

```diff
@@ -91,6 +93,20 @@ def add_ID_variables(
     cps["household_weight"] = household.HSUP_WGT / 1e2
 
 
+def add_personal_variables(cps: h5py.File, person: DataFrame):
+    """Add personal demographic variables.
+
+    Args:
+        cps (h5py.File): The CPS dataset file.
+        person (DataFrame): The CPS person table.
+    """
+    cps["age"] = np.where(
+        person.A_AGE.between(80, 85),
```

Comment:
Does this need to be a `between` statement? I thought it's assigned just one of the values.

Also suggest leaving a comment about what this does and why.

Reply:
To change to `==80`

Maybe assert that there's no raw age between 81 and 84 first?